include/bout/scorepwrapper.hxx

```diff
@@ -47,4 +47,12 @@
 #define SCOREP3(...)
 #endif
 
+/// Instrument a region with scorep
+#ifdef BOUT_HAS_SCOREP
+#define SCOREP_REGION(...)						\
```

Comment:
Apologies for bike-shedding, but it's plausible that `SCOREP_REGION` will clash with a future ScoreP macro/name. We might want to call this `BOUT_SCOREP_REGION` (or similar) instead.

Reply:
np - done.  

PS: I think it's only bike-shedding if I insist on `SCOREP_REGION_BOUT`, we get into a fight about it, people take sides, and consequently v4.3 doesn't get released.